### PR TITLE
Recipe now based on pip 

### PR DESCRIPTION
**Pull request by lmhale99**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by lmhale99** [comment]
Need to rerun tests once atomman finishes updating...

**Comment by lmhale99** [comment]
@jan-janssen,

Can you accept this pull request so that the conda install of iprPy stays up to date?

Lucas

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,61 +1,62 @@
...
-  url: https://github.com/usnistgov/iprPy/archive/95ab94f78826fa3bc94faf2b10dc01477afea762.tar.gz
-  sha256: bd21c51cfd54f366113cf5247f8d2a387262db0a0c98e672233335b52a6f1ca9
-  patches:
-    - iprpydata.patch
-    - python3.patch
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c8b7a78c5dd64c76349d3cdf64120925eda313ef68d4fe0615c9b375846b475a
 
 build:
-  noarch: python
```
Comment (jan-janssen) [inline recipe/meta.yaml:16]: Why is this package no longer `noarch` ?

**Comment by jan-janssen** [comment]
@lmhale99 Sorry for the delay - I had a couple of busy days.

**Comment by lmhale99** [comment]
Not sure, maybe I copied from one of the others. Testing now.

**Comment by lmhale99** [comment]
@jan-janssen fixed!

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Pull request merged**